L1Trigger/L1THGCal/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
 <use   name="Geometry/CaloTopology"/>
 <use   name="Geometry/Records"/>
 <use   name="DataFormats/L1THGCal"/>
+<use   name="SimDataFormats/PileupSummaryInfo"/>
```

Comment:
This would better go in the plugin BuildFile `plugins/BuildFile.xml` in the ntuple section